tests/src/test/java/org/teavm/vm/RecordTest.java

```diff
@@ -74,9 +74,18 @@ public void toStringMethod() {
         assertTrue(index > 0);
     }
 
+    @Test
+    public void toStringMethodPrimitives() {
+        String s = new AllPrimitives(false, (byte) 1, '2', (short) 3, 4, 5L, 6.0f, 7.0, "test").toString();
+        assertEquals("AllPrimitives[a=false, b=1, c=2, s=3, i=4, l=5, f=6.0, d=7.0, o=test]", s);
```

Comment:
I usually don't write record tests like this. There's no specific format in spec, so it may change even across JVM versions, making such tests unstable.

Reply:
Ahh, that's a good point!

I'm not sure it's worth having another version of the existing `toString` test, so could either add all the primitives to the `B` record (and test), or just remove this `toStringMethodPrimitives` test. Do you have any preference?

Reply:
Yes, adding byte and short values to B is a good idea